### PR TITLE
Autotests: #3122 Added three tests of attachment savings

### PR DESCRIPTION
**Pull request by AnastasiyaPiatrovaKlu**
## How the feature works? / How did you fix the issue?
(Screenshots, videos, or GIFs, if applicable)

## Check list
- [ ] unit-tests written
- [ ] e2e-tests written
- [ ] documentation updated
- [x] PR name follows the pattern `#1234 – issue name`
- [x] branch name doesn't contain '#'
- [x] PR is linked with the issue
- [x] base branch (master or release/xx) is correct
- [x] task status changed to "Code review"

### CODE REVIEW

File: ketcher-autotests/tests/Structure-Creating-&-Editing/Actions-With-Structures/Attachment-Point-Tool/attachment-point-tool.spec.ts [hunk 1]
```diff
@@ -339,6 +340,27 @@ test.describe('Attachment Point Tool', () => {
     expect(molFile).toEqual(molFileExpected);
   });
 
+  test('Click and Save as *.mol file', async ({ page }) => {
+    /*
+    Test case: EPMLSOPKET-1747
+    Description: Click the 'Save As' button, and click the 'Save' button.
+    Open the saved *.mol file and edit it in any way.
+    */
+    await openFileAndAddToCanvas('chain-with-attachment-points.ket', page);
```
Comment (Nitvex) [inline ketcher-autotests/tests/Structure-Creating-&-Editing/Actions-With-Structures/Attachment-Point-Tool/attachment-point-tool.spec.ts:349]: I propose to add files to the corresponding folder under `test-data`

File: ketcher-autotests/tests/Structure-Creating-&-Editing/Actions-With-Structures/Attachment-Point-Tool/attachment-point-tool.spec.ts [hunk 2]
```diff
@@ -424,6 +468,30 @@ test.describe('Attachment Point Tool', () => {
     expect(smiFile).toEqual(smiFileExpected);
   });
 
+  test.fixme('Click and Save as *.smi file', async ({ page }) => {
```
Comment (Nitvex) [inline ketcher-autotests/tests/Structure-Creating-&-Editing/Actions-With-Structures/Attachment-Point-Tool/attachment-point-tool.spec.ts:471]: Could you add a comment, why it is marked as `fixme`?

Comment (AnastasiyaPiatrovaKlu) [reply ketcher-autotests/tests/Structure-Creating-&-Editing/Actions-With-Structures/Attachment-Point-Tool/attachment-point-tool.spec.ts:471]: This is a mistake. I will delete it with the next push of changes.

File: ketcher-autotests/tests/Structure-Creating-&-Editing/Actions-With-Structures/Attachment-Point-Tool/attachment-point-tool.spec.ts (hunk 1 above)
Comment (AnastasiyaPiatrovaKlu) [reply ketcher-autotests/tests/Structure-Creating-&-Editing/Actions-With-Structures/Attachment-Point-Tool/attachment-point-tool.spec.ts:349]: Ok. I will do it.

File: ketcher-autotests/tests/Structure-Creating-&-Editing/Actions-With-Structures/Attachment-Point-Tool/attachment-point-tool.spec.ts (hunk 2 above)
Comment (AnastasiyaPiatrovaKlu) [reply ketcher-autotests/tests/Structure-Creating-&-Editing/Actions-With-Structures/Attachment-Point-Tool/attachment-point-tool.spec.ts]: I deleted it.

File: ketcher-autotests/tests/Structure-Creating-&-Editing/Actions-With-Structures/Attachment-Point-Tool/attachment-point-tool.spec.ts (hunk 1 above)
Comment (AnastasiyaPiatrovaKlu) [reply ketcher-autotests/tests/Structure-Creating-&-Editing/Actions-With-Structures/Attachment-Point-Tool/attachment-point-tool.spec.ts]: Done.

**Review by Zhirnoff — APPROVED** [review]

**Review by vitaepam — APPROVED** [review]

**Pull request merged**